admin_manual/configuration/user/user_auth_ldap.rst

```diff
@@ -154,6 +154,27 @@ x users found:
   This is an indicator that tells you approximately how many users will be listed in ownCloud.
   The number updates automatically after any changes.
 
+  Active Directory offers "Recursive retrieval of all AD group memberships of a user". This means essentially that you would be able to search the group you enter and all the other child groups from this groups for users.
+
+  Enter this filter to access this feature::
+
+    ``(&(objectClass=user)(memberof:1.2.840.113556.1.4.1941:=CN=<Group_Name>,DC=example,DC=com))``
+
```

Comment:
Group_Name --> groupname

Reply:
why does it matter?

Reply:
consistency, all the same look professional